spec/rails_helper.rb

```diff
@@ -16,6 +16,7 @@
 WebMock.disable_net_connect!(allow_localhost: true,
   allow: [
     "https://chromedriver.storage.googleapis.com",
+    "https://googlechromelabs.github.io",
```

Comment:
Google added a new Chrome base URL that has been failing Argo builds for two days. This is a separate commit which I added to this PR so the build can pass.